### PR TITLE
Add SLE-15 RT to SP3 branch

### PR DESCRIPTION
**Pull request by JeffreyCheung**

### CODE REVIEW

**Comment by JeffreyCheung** [comment]
This request is to add SLERT15 SP3 base product to Unified Installer in 15 SP3 QU1

**Comment by JeffreyCheung** [comment]
Hi @lslezak , can you please review, this help to add the SLERT15 SP3 product to installer in QU

File: control/control.leanos.xml [hunk 1]
```diff
@@ -166,6 +166,13 @@ textdomain="control"
             <version>15.3</version>
             <register_target>sle-15-$arch</register_target>
           </base_product>
+          <base_product>
+            <display_name>SUSE Linux Enterprise Real Time 15 SP3</display_name>
+            <name>SLE_RT</name>
+            <version>15.3</version>
+            <register_target>sle-15-$arch</register_target>
+            <archs>x86_64</archs>
```
Comment (lslezak) [inline control/control.leanos.xml:174]: So the RT is available only for `x86_64`? (Just double checking...)

File: package/skelcd-control-leanos.changes [hunk 2]
```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Mon Aug  9 13:27:35 UTC 2021 - Jeffrey Cheung <jcheung@suse.com>
+
+- Add RT15 SP3 base product (jsc#SLE-20304)
+- 15.3.9
```
Comment (lslezak) [inline package/skelcd-control-leanos.changes]: For QU we use the `15.3.8.x` versions, `15.3.9` should be used in regular SP3 updates.

Please update also the version also in the `package/skelcd-control-leanos.spec`.

**Review by lslezak — COMMENTED** [review]
Versions

File: control/control.leanos.xml (hunk 1 above)
Comment (JeffreyCheung) [reply control/control.leanos.xml:174]: yes, we currently only support x86_64 platform.

File: package/skelcd-control-leanos.changes (hunk 2 above)
Comment (JeffreyCheung) [reply package/skelcd-control-leanos.changes]: Hi @lslezak , I amended the version to 15.3.8.2 for the 2 files, please check.

**Review by lslezak — APPROVED** [review]
Thank you!

**Comment by lslezak** [comment]
JFYI: Submitted in https://build.suse.de/request/show/252185

**Pull request merged**